### ISSUE TITLE
Liberation serif: Italic symbols for v and greek letter ν (nu) look identical

### ISSUE THREAD

**Issue by isnota-vailable** [liberationfonts/liberation-fonts#60]
**Describe the bug**

Set font to Liberation Serif.
Set to italic.
type v and [ν.]([url](url)) 

**Expected behavior**
They should be easier to distinguish. 
In hydraulics these symbols are often used in the same formulas.
Liberation serif italic is the default formula font in libreoffice. 
I can not use the font because of this.

**Screenshots**
![Bildschirmfoto vom 2023-01-23 15-58-48](https://user-images.githubusercontent.com/102436758/214073244-4e6256d8-402c-4e04-af30-4393ac8c90c6.png)

**Desktop (please complete the following information):**
$ apt show fonts-liberation2 | grep -i version
Version: 2.1.5-1

Libre-office version 7.3.7.2

**Comment by jdpipe** [comment]
Yes, more effort needed to differentiate Greek letters in the Liberation fonts! 
For what it's worth, in LibreOffice, you can re-define symbols in the LibreOffice Math/Formula tool to use a specific font of your choosing, it will be applied when you use %nu (as opposed to typing the unicode using dead_greek)
